MekHQ/src/mekhq/campaign/market/PersonnelMarket.java

```diff
@@ -133,9 +133,7 @@ public void removePersonnelForDay(Campaign c) {
             List<Person> toRemove = method.removePersonnelForDay(c, personnel);
             if (null != toRemove) {
                 for (Person p : toRemove) {
-                	if (attachedEntities.contains(p.getId())) {
```

Comment:
`Hashmap.contains()` is the same as `Map.containsValue()`.
Since here it's called with a key, the condition is always false.
(this was reported by findbugs)

Calling `remove()` with a non-contained key is a nop, so I just removed the `if` altogether.